### DISCUSSION TITLE
[Feature]: Separate settings for Thumbnail and Original photo storage locations in .env file

### DISCUSSION THREAD

**Discussion by jammin1911** [immich-app/immich#1685] [Feature Request]
### Feature detail

Separate storage path specification in .env for where to store thumbnails (fast ssd or other local storage) and original photos (slow/less expensive storage or cloud drive).

### Platform

Server

**Comment by dhruvinsh** [comment]
Yes this would be good feature, A homeserver like unraid would definately benifit from it. Main user media (original photos) stays on array (or raid/nas drive) nad thumbnails stays on the some cache (usually faster SSD) folder.

**Comment by panoti** [comment]
Hi guys, let me close this feature issue because of https://github.com/immich-app/immich/pull/488#issuecomment-1242629107. If there is anyone really need this feature, you can use my images `ghcr.io/panoti/immich/backend:latest` and `ghcr.io/panoti/immich/ml-worker:latest` for workaround, thanks all.

**Comment by alextran1502** [comment]
@panoti it was the implementation of the PR. I would still like to have this though

**Comment by panoti** [comment]
@alextran1502 really? I'll wait for the better implementation.

**Comment by jesvinjoachim** [comment]
any progress on this?

**Comment by jesvinjoachim** [comment]
we already have the folder structure(sorting images by date and month for images as a specific folder ) and it seems, we can do this on the same ui page itself. i manually tried the path of the thumb and it works flawlessly .. on a previous version.

**Comment by bo0tzz** [comment]
Added by #2047

**Reply by klejejs** [reply]
@bo0tzz Is there any documentation on how this could be applied in deployment? Unfortunately, I couldn't find anything.

**Reply by bo0tzz** [reply]
There's not really any documentation. You would just mount the different folders to different storage depending on your needs.
